### PR TITLE
Wallet Redesign and Theme update

### PR DESCRIPTION
**Pull request by avoocado-dev**

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -0,0 +1,4 @@
+REACT_APP_CHAIN_ID="1"
+NEXT_PUBLIC_REACT_APP_INFURA_ID=4ec7cc5b5f684e3fba7ba2e0c1533d91
+NEXT_PUBLIC_OPENSEA_API_KEY=83be70605c2441f399b283690a29ee8c
+NEXT_PUBLIC_DEV_MODE=wat
```
Comment (just-toby) [inline .env.production]: please remove this and add this file to gitignore. we set these for production via vercel, and we don't want them publicly visible.

File: src/components/AccountDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default function AccountDetails(props) {
```
Comment (just-toby) [inline src/components/AccountDetails/index.tsx]: remove this file?

Comment (avoocado-dev) [reply src/components/AccountDetails/index.tsx]: Lololol forgot about that my bad

File: .env.production (hunk 1 above)
Comment (avoocado-dev) [reply .env.production]: fasho fasho

**Comment by avoocado-dev** [comment]
Atm metamask and wallet connect are the two available options. I'll work on adding the others in a later update.

File: .env.production [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/4bf032f2d38a4ed6bb975b80d6340847"
```
Comment (just-toby) [inline .env.production]: remove this file please

File: src/components/Header/HeaderActions.tsx [hunk 4]
```diff
@@ -33,7 +39,23 @@ function HeaderActions (props) {
         <div className="flexStretch"/>
 
         <div className="navPadding"> 
-            <HeaderWalletConnector/>
+            {account ? (
+                <Button 
+                    // href="/"
+                    onClick={toggleWalletDropdown}
```
Comment (just-toby) [inline src/components/Header/HeaderActions.tsx:45]: what do you think about making this disconnect the wallet? that was a request from one of the other guys in the group, that you be able to disconnect in the UI vs requiring you to use metamask to disconnect

File: src/connectors/index.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
+import { WalletLinkConnector } from '@web3-react/walletlink-connector'
+import { PortisConnector } from '@web3-react/portis-connector'
+
+import { FortmaticConnector } from './Formatic'
+import { NetworkConnector } from './NetworkConnector'
+
+const NETWORK_URL = process.env.REACT_APP_NETWORK_URL
+const FORMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY
+const PORTIS_ID = process.env.REACT_APP_PORTIS_ID
```
Comment (just-toby) [inline src/connectors/index.tsx]: these need to start with NEXT_PUBLIC_ to work in the browser

**Review by just-toby — APPROVED** [review]

File: src/connectors/index.tsx (hunk 5 above)
Comment (avoocado-dev) [reply src/connectors/index.tsx]: those consts weren't being used so just set them to nothing for right now. Once I set up coinbase, formatic, and portis I'll come back to this

File: .env.production (hunk 3 above)
Comment (avoocado-dev) [reply .env.production]: done

**Pull request merged**